cloud/azure/actuators/machine/machineactuator.go

```diff
@@ -123,11 +122,18 @@ func (azure *AzureClient) Create(cluster *clusterv1.Cluster, machine *clusterv1.
 	if err != nil {
 		return err
 	}
-	// TODO: Set up Kubernetes
+	if machine.ObjectMeta.Annotations == nil {
+		machine.ObjectMeta.Annotations = make(map[string]string)
+	}
+	if azure.v1Alpha1Client != nil {
+		machine.ObjectMeta.Annotations["dummy"] = "dummy"
```

Comment:
Is this still necessary? I thought this was for testing.

Reply:
The Cluster API currently determines that a machine's been created/updated by whether an annotation has been set, so rather than "dummy", some useful information could be put there instead, like the machine name. Changing to include some relevant info.